src/node_crypto.cc

```diff
@@ -1027,6 +1027,38 @@ static X509_STORE* NewRootCertStore() {
 }
 
 
+X509_STORE* CloneRootCertStore() {
+  if (root_cert_store == nullptr) {
+    return NewRootCertStore();
+  }
```

Comment:
Minor nit that's worthy of debate ;-) ...
 
```suggestion
  if (root_cert_store == nullptr)
    return NewRootCertStore();
```